lib/types.js

```diff
@@ -2,11 +2,44 @@ const isNumber = val => typeof val === 'number';
 
 const castToNumber = val => {
   if(isNumber(val)) return val;
+
   const number = Number(val);
-  if(isNaN(number)) throw new CastError(Number, val);
+  if(isNaN(number)) { throw new CastError(Number, val);
```

Comment:
if you're going to use a bracket '{' here  then put the throw statement on a new line. or remove brackets ` if(isNaN(number)) throw new CastError(Number, val);`